openstudiocore/src/utilities/core/FileReference.cpp

```diff
@@ -150,7 +150,7 @@ void FileReference::setPath(const openstudio::path& newPath) {
 bool FileReference::makePathAbsolute(const openstudio::path& searchDirectory) {
```

Comment:
FileReference should be deprecated, I think it was mostly used in the project database